source/includes/User/user.md

```diff
@@ -8,18 +8,18 @@ The real-life person is represented by a Person record in OpenMRS, and a person
 
 ## Available operations for User
 
-1. [List users](#list-all-non-retired-users)
+1. [List users](#list-all-users)
 2. [Create a user](#create-a-user)
 3. [Update a user](#update-a-user)
 4. [Delete a user](#delete-a-user)
 
 
-## List all non-retired users.
```

Comment:
Does this mean that the `non-retired` users was wrong?

Reply:
I've removed `non-retired` because actually you can query non-retired or including retired. Depends on the value of the parameter passed

Reply:
I thought that the documentation is about what is returned by default. non-retired users. Then if one wants the non default (retired users), they add that parameter.

Reply:
I've updated the PR :)